.travis.yml

```diff
@@ -11,12 +11,16 @@ env:
     - BUILD_GOOS=darwin
     - BUILD_GOOS=windows
 install:
-- go get golang.org/x/tools/cmd/vet
-- go get -v github.com/zquestz/s
+  - go get golang.org/x/tools/cmd/vet
+  - go get -v github.com/zquestz/s
 script:
-- go build
-- go fmt ./...
-- go vet ./...
-- go test -i -race ./...
-- go test -v -race ./...
-
+  - go build
+  - go fmt ./...
+  - go vet ./...
+  - go test -i -race ./...
+  - go test -v -race ./...
+after_script:
+  - if [ "$TRAVIS_GO_VERSION" = "1.5.3" ] && [ "$BUILD_GOOS" = "linux" ] && [ "$TRAVIS_TAG" != "" ]; then go get github.com/inconshreveable/mousetrap; fi
```

Comment:
Why do we need the same condition multiple times? Can't we dry this up and have it check that condition just once and then do all the go get's we need?